test/test-eth-ledger-bridge-keyring.js

```diff
@@ -47,10 +49,19 @@ chai.use(spies)
 describe('LedgerBridgeKeyring', function () {
 
   let keyring
+  let sandbox
 
-  beforeEach(function () {
+  beforeEach(async function () {
+    sandbox = chai.spy.sandbox()
     keyring = new LedgerBridgeKeyring()
     keyring.hdk = fakeHdKey
+    keyring.setAccountToUnlock(fakeAccounts[0])
```

Comment:
This looks like a mistake. This method is supposed to take the _index_ of the account that you want to be unlocked, and it is initialized at 0 anyway, which is what it seems like you're trying to set it to.

Right now you're passing in the _address_ and it's being parsed as a _decimal_ number, as an index.

Reply:
Thanks, fixed in latest.